<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBusTest</class>
 <widget class="QMainWindow" name="DBusTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBusTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Connection</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QRadioButton" name="rbServer">
         <property name="text">
          <string>Run Server</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="btnConnect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="rbSystemBus">
         <property name="text">
          <string>System Bus</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rbSessionBus">
         <property name="text">
          <string>Session Bus</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="rbPeer">
         <property name="text">
          <string>Peer Connection</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="rbPrivateBus">
         <property name="text">
          <string>Private Bus</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="txtPrivateBus">
         <property name="text">
          <string>tcp:host=127.0.0.1,port=54320</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QLineEdit" name="txtPeer">
         <property name="text">
          <string>tcp:host=127.0.0.1,port=55555</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QLineEdit" name="txtServer">
         <property name="text">
          <string>tcp:host=127.0.0.1,port=55555</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLabel" name="lblConnStatus">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Consumer</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="cbConsume">
           <property name="text">
            <string>Consume Object</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtConsPath">
           <property name="text">
            <string>/Vehicle</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lblConsStatus">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="btnConsSpeed">
           <property name="minimum">
            <number>-1000000</number>
           </property>
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnConsAcc">
           <property name="text">
            <string>Accelerate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnConsDec">
           <property name="text">
            <string>Decelerate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnConsSet">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numConsSet">
           <property name="minimum">
            <number>-1000000</number>
           </property>
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Provider</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="cbProvide">
           <property name="text">
            <string>Provide Object</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtProvPath">
           <property name="text">
            <string>/Vehicle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Delay:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numDelay">
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="singleStep">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lblProvStatus">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QSpinBox" name="numProvSpeed">
           <property name="minimum">
            <number>-1000000</number>
           </property>
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnProvAcc">
           <property name="text">
            <string>Accelerate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnProvDec">
           <property name="text">
            <string>Decelerate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnProvSet">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numProvSet">
           <property name="minimum">
            <number>-1000000</number>
           </property>
           <property name="maximum">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnTestCon">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTestDisconn">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
